UnitTests/BraintreeTestShared/MockViewControllerPresentingDelegate.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+import BraintreeCore
+
+@objc public class MockViewControllerPresentingDelegate : NSObject, BTViewControllerPresentingDelegate {
```

Comment:
This originally lived in `MockDelegates.swift`, which we renamed to `MockViewControllerPresentingDelegate.swift`. After deleting the `MockAppSwitchDelegate`, there was only one class left in the file, so it made sense for the file name to match the class.